pkg/crc/services/dns/dns.go

```diff
@@ -49,10 +49,10 @@ func setupDnsmasq(serviceConfig services.ServicePostStartConfig) error {
 	}
 
 	// Remove the dnsmasq container if it exists during the VM stop cycle
-	_, _, _ = serviceConfig.SSHRunner.Run("sudo podman rm -f dnsmasq")
+	_, _, _ = serviceConfig.SSHRunner.Run("sudo podman rm -f crc-dnsmasq")
 
 	// Start the dnsmasq container
-	dnsServerRunCmd := fmt.Sprintf("sudo podman run  --ip %s --name dnsmasq -v %s:/etc/dnsmasq.conf -p 53:%d/udp --privileged -d %s",
+	dnsServerRunCmd := fmt.Sprintf("sudo podman run  --ip %s --name crc-dnsmasq -v %s:/etc/dnsmasq.conf -p 53:%d/udp --privileged -d %s",
```

Comment:
this commit (` Rename dnsmasq container to crc-dnsmasq `)  can be removed (squashed with the next commit) as its code will be gone after next commit.